plugins/out_stackdriver/stackdriver.c

```diff
@@ -524,6 +637,147 @@ static int stackdriver_format(const void *data, size_t bytes,
       msgpack_pack_str_body(&mp_pck,
                             ctx->instance_id, flb_sds_len(ctx->instance_id));
     }
+    else if (strcmp(ctx->resource, "k8s_container") == 0) {
+      /* k8s_container resource has fields project_id, location, cluster_name,
```

Comment:
Add comment to indicate the value of k8s_container resource type in local_resource id should be like "k8s_container.<namespace_name>.<pod_name>.<container_name>"